functions.lua

```diff
@@ -303,41 +303,50 @@ travelnet.rotate_player = function(target_pos, player)
 end
 
 
-travelnet.remove_box = function(_, _, oldmetadata, digger)
+travelnet.remove_box_action = function(oldmetadata)
 	if not oldmetadata or oldmetadata == "nil" or not oldmetadata.fields then
-		minetest.chat_send_player(digger:get_player_name(), S("Error") .. ": " ..
-				S("Could not find information about the station that is to be removed."))
-		return
+		return false, S("Could not find information about the station that is to be removed.")
 	end
 
-	local owner_name      = oldmetadata.fields["owner_name"]
+	local owner_name      = oldmetadata.fields["owner"]
 	local station_name    = oldmetadata.fields["station_name"]
 	local station_network = oldmetadata.fields["station_network"]
 
 	-- station is not known? then just remove it
 	if	not (owner_name and station_network and station_name)
 		or not travelnet.get_station(owner_name, station_network, station_name)
 	then
-		minetest.chat_send_player(digger:get_player_name(), S("Error") .. ": " ..
-				S("Could not find the station that is to be removed."))
-		return
+		return false, S("Could not find the station that is to be removed.")
```

Comment:
Shouldn't this `return true, S("Could not find the station that is to be removed.")` to remove node but just without touching travelnet db?

Reply:
Now after reading PR comments seems like you were already thinking exact same thing 😄

Reply:
I think it should return false on failure, but the logic that responds to it needs to be changed. Just making sure that logic makes sense to others, thanks for responding

Reply:
That should work as before now, the high-level remove action has been reverted, this is now just a correction and some neatening